cvs server: Diffing .
Index: gamewidget.cpp
===================================================================
RCS file: /home/kde/kdegames/ksmiletris/gamewidget.cpp,v
retrieving revision 1.12
diff -u -u -p -r1.12 gamewidget.cpp
--- kdegames/ksmiletris/gamewidget.cpp	2000/09/02 03:39:44	1.12
+++ kdegames/ksmiletris/gamewidget.cpp	2002/03/08 10:32:50
@@ -88,6 +88,7 @@ GameWidget::~GameWidget()
 	delete [] sprites;
 	delete [] map;
 	delete [] mirror_sprites;
+	delete timer;
 }
 
 void GameWidget::playSound(Sound s)
Index: gamewindow.cpp
===================================================================
RCS file: /home/kde/kdegames/ksmiletris/gamewindow.cpp,v
retrieving revision 1.19
diff -u -u -p -r1.19 gamewindow.cpp
--- kdegames/ksmiletris/gamewindow.cpp	2001/06/08 18:07:02	1.19
+++ kdegames/ksmiletris/gamewindow.cpp	2002/03/08 10:32:50
@@ -115,6 +115,15 @@ GameWindow::GameWindow(QWidget *, const 
 	game->do_sounds = config->readBoolEntry("Sounds", true);
 	options_popup->setItemChecked(soundsID, game->do_sounds);
 }
+#include <kdebug.h>
+GameWindow::~GameWindow()
+{
+  delete game;
+  delete options_popup;
+  delete menu;
+  delete status;
+  delete pieces_popup;  
+}
 
 void GameWindow::menu_newGame()
 {
Index: gamewindow.h
===================================================================
RCS file: /home/kde/kdegames/ksmiletris/gamewindow.h,v
retrieving revision 1.6
diff -u -u -p -r1.6 gamewindow.h
--- kdegames/ksmiletris/gamewindow.h	2000/09/02 03:39:44	1.6
+++ kdegames/ksmiletris/gamewindow.h	2002/03/08 10:32:50
@@ -38,7 +38,7 @@ class GameWindow : public KMainWindow {
 
 public:
 	GameWindow(QWidget *parent=0, const char *name=0);
-
+	~GameWindow();
 private:
 	enum menuPosition { Top, Bottom, Floating };	// hack
 	KMenuBar *menu;
cvs server: Diffing data
cvs server: Diffing wav
