--- kdegames-3.0.3/kwin4/kwin4/kspritecache.cpp--	2002-08-26 10:42:27.000000000 +0200
+++ kdegames-3.0.3/kwin4/kwin4/kspritecache.cpp	2002-08-26 10:43:03.000000000 +0200
@@ -29,6 +29,7 @@ KSpriteCache::~KSpriteCache()
   kdDebug(11002) << "KSpriteCache: ItemDict=" << mItemDict.count() << endl; 
   kdDebug(11002) << "KSpriteCache: CloneDict=" << mCloneDict.count() << endl; 
   reset();
+  delete mConfig;
 }
 
 void KSpriteCache::setRcFile(QString name)
