cvs server: Diffing .
Index: game.cpp
===================================================================
RCS file: /home/kde/kdegames/kjezz/Attic/game.cpp,v
retrieving revision 1.8
diff -u -u -p -r1.8 game.cpp
--- kdegames/kjezz/game.cpp	2001/01/13 14:07:38	1.8
+++ kdegames/kjezz/game.cpp	2002/03/07 13:31:26
@@ -419,6 +419,7 @@ JezzGame::~JezzGame()
     delete m_field;
     delete m_ballPixmaps;
     delete m_artsServer;
+    delete m_clock;
 }
 
 
cvs server: Diffing pics
cvs server: Diffing sounds
