cvs server: Diffing .
cvs server: Diffing kbattleship
Index: kbattleship/kclientdialog.cpp
===================================================================
RCS file: /home/kde/kdegames/kbattleship/kbattleship/kclientdialog.cpp,v
retrieving revision 1.20
diff -u -p -r1.20 kclientdialog.cpp
--- kdegames/kbattleship/kbattleship/kclientdialog.cpp	2001/06/11 21:19:28	1.20
+++ kdegames/kbattleship/kbattleship/kclientdialog.cpp	2002/03/04 14:41:59
@@ -27,6 +27,7 @@ KClientDialog::KClientDialog(QWidget *pa
 	connect(connectBtn, SIGNAL(clicked()), this, SLOT(slotConnectClicked()));
 	connect(cancelBtn, SIGNAL(clicked()), this, SLOT(slotCancelClicked()));
 	connect(serverEdit, SIGNAL(returnPressed(const QString &)), this, SLOT(slotReturnPressed(const QString &)));
+	connect(serverEdit->lineEdit(), SIGNAL(textChanged ( const QString & )), this, SLOT(slotServerChanged(const QString &)));
 	m_config->setGroup("History");
 	serverEdit->completionObject()->setItems(m_config->readListEntry("CompletionList")); 
 
@@ -34,6 +35,7 @@ KClientDialog::KClientDialog(QWidget *pa
 	serverEdit->setHistoryItems(m_config->readListEntry("HistoryList"));
 
 	serverEdit->setCurrentItem(m_config->readNumEntry("Index", -1));
+	connectBtn->setEnabled(!serverEdit->lineEdit()->text().isEmpty());
 }
 
 KClientDialog::~KClientDialog()
@@ -43,6 +45,11 @@ KClientDialog::~KClientDialog()
 	m_config->writeEntry("HistoryList", serverEdit->historyItems());
 	m_config->writeEntry("Index", serverEdit->currentItem());
 	m_config->sync();
+}
+
+void KClientDialog::slotServerChanged(const QString &server)
+{
+  connectBtn->setEnabled(!server.isEmpty());
 }
 
 void KClientDialog::slotConnectClicked()
Index: kbattleship/kclientdialog.h
===================================================================
RCS file: /home/kde/kdegames/kbattleship/kbattleship/kclientdialog.h,v
retrieving revision 1.16
diff -u -p -r1.16 kclientdialog.h
--- kdegames/kbattleship/kbattleship/kclientdialog.h	2001/06/11 21:19:28	1.16
+++ kdegames/kbattleship/kbattleship/kclientdialog.h	2002/03/04 14:41:59
@@ -42,6 +42,7 @@ public slots:
 	void slotConnectClicked();
 	void slotCancelClicked();
 	void slotReturnPressed(const QString &hostname);
+	void slotServerChanged(const QString &server);
 
 signals:
 	void sigConnectServer();
cvs server: Diffing kbattleship/dialogs
cvs server: Diffing kbattleship/pictures
cvs server: Diffing kbattleship/sounds
