cvs server: Diffing .
Index: AbTop.cpp
===================================================================
RCS file: /home/kde/kdegames/kabalone/Attic/AbTop.cpp,v
retrieving revision 1.43
diff -u -u -p -r1.43 AbTop.cpp
--- kdegames/kabalone/AbTop.cpp	2001/05/03 13:24:31	1.43
+++ kdegames/kabalone/AbTop.cpp	2002/03/07 12:48:41
@@ -107,6 +107,7 @@ AbTop::~AbTop()
   /* Unregister from other abalone processes */
   if (net)
     delete net;
+  delete timer;
 
 #ifdef SPION
   delete spy;
Index: BoardWidget.cpp
===================================================================
RCS file: /home/kde/kdegames/kabalone/Attic/BoardWidget.cpp,v
retrieving revision 1.17
diff -u -u -p -r1.17 BoardWidget.cpp
--- kdegames/kabalone/BoardWidget.cpp	2001/05/03 13:24:31	1.17
+++ kdegames/kabalone/BoardWidget.cpp	2002/03/07 12:48:42
@@ -83,6 +83,11 @@ BoardWidget::~BoardWidget()
     if (arrow[i] != 0)
       delete arrow[i];
 
+  delete boardColor;
+  delete redColor;
+  delete yellowColor;
+  delete redHColor;
+  delete yellowHColor;
 #ifdef HAVE_KIR  
   if (m_backRenderer != 0)
     delete m_backRenderer;
cvs server: Diffing bitmaps
cvs server: Diffing toolbar
