cvs server: Diffing .
Index: pwidget.cpp
===================================================================
RCS file: /home/kde/kdegames/kpat/pwidget.cpp,v
retrieving revision 1.74
diff -u -p -r1.74 pwidget.cpp
--- kdegames/kpat/pwidget.cpp	2001/06/23 22:34:45	1.74
+++ kdegames/kpat/pwidget.cpp	2002/03/04 10:11:41
@@ -200,11 +200,11 @@ void pWidget::changeWallpaper()
     if (dill) {
         KConfig *config = kapp->config();
         KConfigGroupSaver kcs(config, settings_group);
-
+	/*
         QString deck = config->readEntry("Back");
         QString dummy = config->readEntry("Cards");
         setBackSide(deck, dummy);
-
+	*/
 	config->writeEntry("Background", bgpath);
         dill->setBackgroundPixmap(background, midcolor);
         dill->canvas()->setAllChanged();
cvs server: Diffing freecell-solver
cvs server: Diffing freecell-solver/board_gen
cvs server: Diffing icons
