diff -urbB kdegames-3.4.2.org/kfouleggs/Makefile.am kdegames-3.4.2/kfouleggs/Makefile.am
--- kdegames-3.4.2.org/kfouleggs/Makefile.am	2005-07-20 10:00:23.000000000 +0000
+++ kdegames-3.4.2/kfouleggs/Makefile.am	2005-07-23 12:06:08.390563920 +0000
@@ -25,44 +25,4 @@
 DESTHIGHSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)
 DESTSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)/$(bin_PROGRAMS).scores
 
-install-data-local:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-	&& echo "********************************************************" \
-	&& echo "" \
-	&& echo "This game is installed sgid \"games\" to use the" \
-	&& echo "system-wide highscore file (in "$(HIGHSCORE_DIRECTORY)")." \
-	&& echo "" \
-	&& echo "If the system-wide highscore file does not exist, it is" \
-        && echo "created with the correct ownership and permissions. See the" \
-        && echo "INSTALL file in \"kdegames/libkdegames/highscore\" for details." \
-	&& echo "" \
-	&& echo "********************************************************" \
-	) || true
 
-install-exec-hook:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-		&& ((mkdir -p $(DESTHIGHSCORES) && chown $(highscore_user):$(highscore_group) $(DESTHIGHSCORES) \
-		&& chmod 750 $(DESTHIGHSCORES)) \
-	|| echo "Error: Could not create the highscore directory with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test ${setgid} = true \
-        && ((chmod 2755  $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((touch $(DESTSCORES) && chown $(highscore_user):$(highscore_group) $(DESTSCORES) \
-	&& chmod 0660 $(DESTSCORES)) \
-	|| echo "Error: Could not create system-wide highscore file with correct permissions !!" \
-	)) || true
diff -urbB kdegames-3.4.2.org/klickety/Makefile.am kdegames-3.4.2/klickety/Makefile.am
--- kdegames-3.4.2.org/klickety/Makefile.am	2005-07-20 10:00:21.000000000 +0000
+++ kdegames-3.4.2/klickety/Makefile.am	2005-07-23 12:05:08.278702312 +0000
@@ -25,44 +25,3 @@
 DESTHIGHSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)
 DESTSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)/$(bin_PROGRAMS).scores
 
-install-data-local:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-	&& echo "********************************************************" \
-	&& echo "" \
-	&& echo "This game is installed sgid \"games\" to use the" \
-	&& echo "system-wide highscore file (in "$(HIGHSCORE_DIRECTORY)")." \
-	&& echo "" \
-	&& echo "If the system-wide highscore file does not exist, it is" \
-        && echo "created with the correct ownership and permissions. See the" \
-        && echo "INSTALL file in \"kdegames/libkdegames/highscore\" for details." \
-	&& echo "" \
-	&& echo "********************************************************" \
-	) || true
-
-install-exec-hook:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-		&& ((mkdir -p $(DESTHIGHSCORES) && chown $(highscore_user):$(highscore_group) $(DESTHIGHSCORES) \
-		&& chmod 750 $(DESTHIGHSCORES)) \
-	|| echo "Error: Could not create the highscore directory with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test ${setgid} = true \
-        && ((chmod 2755  $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((touch $(DESTSCORES) && chown $(highscore_user):$(highscore_group) $(DESTSCORES) \
-	&& chmod 0660 $(DESTSCORES)) \
-	|| echo "Error: Could not create system-wide highscore file with correct permissions !!" \
-	)) || true
diff -urbB kdegames-3.4.2.org/kmines/Makefile.am kdegames-3.4.2/kmines/Makefile.am
--- kdegames-3.4.2.org/kmines/Makefile.am	2005-07-20 10:00:23.000000000 +0000
+++ kdegames-3.4.2/kmines/Makefile.am	2005-07-23 12:05:56.938304928 +0000
@@ -20,45 +20,3 @@
 DESTHIGHSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)
 DESTSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)/$(bin_PROGRAMS).scores
 
-install-data-local:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-	&& echo "********************************************************" \
-	&& echo "" \
-	&& echo "This game is installed sgid \"games\" to use the" \
-	&& echo "system-wide highscore file (in "$(HIGHSCORE_DIRECTORY)")." \
-	&& echo "" \
-	&& echo "If the system-wide highscore file does not exist, it is" \
-        && echo "created with the correct ownership and permissions. See the" \
-        && echo "INSTALL file in \"kdegames/libkdegames/highscore\" for details." \
-	&& echo "" \
-	&& echo "********************************************************" \
-	) || true
-
-install-exec-hook:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-		&& ((mkdir -p $(DESTHIGHSCORES) && chown $(highscore_user):$(highscore_group) $(DESTHIGHSCORES) \
-		&& chmod 750 $(DESTHIGHSCORES)) \
-	|| echo "Error: Could not create the highscore directory with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test ${setgid} = true \
-        && ((chmod 2755  $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((touch $(DESTSCORES) && chown $(highscore_user):$(highscore_group) $(DESTSCORES) \
-	&& chmod 0660 $(DESTSCORES)) \
-	|| echo "Error: Could not create system-wide highscore file with correct permissions !!" \
-	)) || true
-
diff -urbB kdegames-3.4.2.org/ksirtet/ksirtet/Makefile.am kdegames-3.4.2/ksirtet/ksirtet/Makefile.am
--- kdegames-3.4.2.org/ksirtet/ksirtet/Makefile.am	2005-07-20 10:00:21.000000000 +0000
+++ kdegames-3.4.2/ksirtet/ksirtet/Makefile.am	2005-07-23 12:05:43.798302512 +0000
@@ -30,44 +30,3 @@
 DESTHIGHSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)
 DESTSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)/$(bin_PROGRAMS).scores
 
-install-data-local:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-	&& echo "********************************************************" \
-	&& echo "" \
-	&& echo "This game is installed sgid \"games\" to use the" \
-	&& echo "system-wide highscore file (in "$(HIGHSCORE_DIRECTORY)")." \
-	&& echo "" \
-	&& echo "If the system-wide highscore file does not exist, it is" \
-        && echo "created with the correct ownership and permissions. See the" \
-        && echo "INSTALL file in \"kdegames/libkdegames/highscore\" for details." \
-	&& echo "" \
-	&& echo "********************************************************" \
-	) || true
-
-install-exec-hook:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-		&& ((mkdir -p $(DESTHIGHSCORES) && chown $(highscore_user):$(highscore_group) $(DESTHIGHSCORES) \
-		&& chmod 750 $(DESTHIGHSCORES)) \
-	|| echo "Error: Could not create the highscore directory with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test ${setgid} = true \
-        && ((chmod 2755  $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((touch $(DESTSCORES) && chown $(highscore_user):$(highscore_group) $(DESTSCORES) \
-	&& chmod 0660 $(DESTSCORES)) \
-	|| echo "Error: Could not create system-wide highscore file with correct permissions !!" \
-	)) || true
