diff -urbB kdegames-3.5.0.org/kbounce/Makefile.am kdegames-3.5.0/kbounce/Makefile.am
--- kdegames-3.5.0.org/kbounce/Makefile.am	2005-11-08 22:24:52.000000000 +0000
+++ kdegames-3.5.0/kbounce/Makefile.am	2005-11-24 17:13:07.068864952 +0000
@@ -31,45 +31,3 @@
 DESTHIGHSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)
 DESTSCORES = $(DESTDIR)$(HIGHSCORE_DIRECTORY)/$(bin_PROGRAMS).scores
 
-install-data-local:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-	&& echo "********************************************************" \
-	&& echo "" \
-	&& echo "This game is installed sgid \"games\" to use the" \
-	&& echo "system-wide highscore file (in "$(HIGHSCORE_DIRECTORY)")." \
-	&& echo "" \
-	&& echo "If the system-wide highscore file does not exist, it is" \
-        && echo "created with the correct ownership and permissions. See the" \
-        && echo "INSTALL file in \"kdegames/libkdegames/highscore\" for details." \
-	&& echo "" \
-	&& echo "********************************************************" \
-	) || true
-
-install-exec-hook:
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-		&& ((mkdir -p $(DESTHIGHSCORES) && chown $(highscore_user):$(highscore_group) $(DESTHIGHSCORES) \
-		&& chmod 750 $(DESTHIGHSCORES)) \
-	|| echo "Error: Could not create the highscore directory with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((chown $(highscore_user):$(highscore_group) $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test ${setgid} = true \
-        && ((chmod 2755  $(DESTBIN)) \
-	|| echo "Error: Could not install the game with correct permissions !!" \
-	)) || true
-
-	@(test x$(HIGHSCORE_DIRECTORY) != x \
-        && ((touch $(DESTSCORES) && chown $(highscore_user):$(highscore_group) $(DESTSCORES) \
-	&& chmod 0660 $(DESTSCORES)) \
-	|| echo "Error: Could not create system-wide highscore file with correct permissions !!" \
-	)) || true
-
