diff -dur kdegames-1.0.orig/ksnake/data/Makefile.am kdegames-1.0/ksnake/data/Makefile.am
--- kdegames-1.0.orig/ksnake/data/Makefile.am	Sat Jun 27 11:29:59 1998
+++ kdegames-1.0/ksnake/data/Makefile.am	Sun Oct 11 16:05:05 1998
@@ -1,6 +1,6 @@
 
 
-highscoredir = $(kde_datadir)/ksnake
+highscoredir = $(prefix)/../../var/state/games/ksnake
 highscore_DATA = highScores
 
 SUBDIRS = levels pixmaps backgrounds
diff -dur kdegames-1.0.orig/ksnake/data/Makefile.in kdegames-1.0/ksnake/data/Makefile.in
--- kdegames-1.0.orig/ksnake/data/Makefile.in	Fri Jul 10 20:36:15 1998
+++ kdegames-1.0/ksnake/data/Makefile.in	Sun Oct 11 16:04:47 1998
@@ -134,7 +134,7 @@
 x_includes = @x_includes@
 x_libraries = @x_libraries@
 
-highscoredir = $(kde_datadir)/ksnake
+highscoredir = $(prefix)/../../var/state/games/ksnake
 highscore_DATA = highScores
 
 SUBDIRS = levels pixmaps backgrounds
diff -dur kdegames-1.0.orig/ksnake/score.cpp kdegames-1.0/ksnake/score.cpp
--- kdegames-1.0.orig/ksnake/score.cpp	Sun Jun 14 22:19:53 1998
+++ kdegames-1.0/ksnake/score.cpp	Sun Oct 11 16:05:47 1998
@@ -20,7 +20,7 @@
     :QObject( parent, name )
 {
     QString libDir;
-    libDir.setStr(KApplication::kde_datadir().copy());
+    libDir.setStr("/var/state/games");
     libDir.append("/ksnake/");
     file.setName( libDir + "highScores");
     playerName = getenv("LOGNAME");
