cvs server: Diffing .
Index: brain.cpp
===================================================================
RCS file: /home/kde/kdegames/kjumpingcube/brain.cpp,v
retrieving revision 1.5
diff -u -u -p -r1.5 brain.cpp
--- kdegames/kjumpingcube/brain.cpp	2001/04/12 15:54:46	1.5
+++ kdegames/kjumpingcube/brain.cpp	2002/03/07 13:49:36
@@ -292,7 +292,7 @@ double Brain::doMove(int row, int column
 	    worth+=doMove(c2m[i].row,c2m[i].column,player,box);
          }
       }
-
+      delete c2m;
       currentLevel--;
       return worth;
 
