Index: kscoredialog.cpp
===================================================================
RCS file: /home/kde/kdegames/libkdegames/kscoredialog.cpp,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -u -p -r1.5 -r1.6
--- kdegames-3.0.3/kdegames/libkdegames/kscoredialog.cpp	2002/02/09 01:26:28	1.5
+++ kdegames-3.0.3/kdegames/libkdegames/kscoredialog.cpp	2002/09/10 04:27:22	1.6
@@ -29,6 +29,7 @@ this software.
 #include <qlineedit.h>
 #include <qwidgetstack.h>
 #include <qtimer.h>
+#include <qevent.h>
 #include <qptrvector.h>
 
 #include <kapplication.h>
@@ -85,6 +86,8 @@ KScoreDialog::KScoreDialog(int fields, Q
    d->header[Score] = i18n("Score");
    d->key[Score] = "Score";
    d->page = makeMainWidget();
+   
+   connect(this, SIGNAL(okClicked()), SLOT(slotGotName()));
 }
 
 KScoreDialog::~KScoreDialog()
@@ -235,7 +238,6 @@ void KScoreDialog::aboutToShow()
            d->edit->setFocus();
            connect(d->edit, SIGNAL(returnPressed()), 
                  this, SLOT(slotGotReturn()));
-           enableButtonOK(false);
          }
          else
          {
@@ -382,7 +384,6 @@ void KScoreDialog::slotGotName()
    delete d->edit;
    d->edit = 0;
    d->newName = -1;
-   enableButtonOK(true);
 }
 
 int KScoreDialog::highScore()
@@ -391,6 +392,16 @@ int KScoreDialog::highScore()
       loadScores();
 
    return (*d->scores.first())[Score].toInt();
+}
+
+void KScoreDialog::keyPressEvent( QKeyEvent *ev)
+{
+   if ((d->newName != -1) && (ev->key() == Key_Return))
+   {
+       ev->ignore();
+       return;
+   }
+   KDialogBase::keyPressEvent(ev);
 }
 
 
Index: kscoredialog.h
===================================================================
RCS file: /home/kde/kdegames/libkdegames/kscoredialog.h,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -u -p -r1.5 -r1.6
--- kdegames-3.0.3/kdegames/libkdegames/kscoredialog.h	2001/12/25 07:06:26	1.5
+++ kdegames-3.0.3/kdegames/libkdegames/kscoredialog.h	2002/09/10 04:27:22	1.6
@@ -111,6 +111,7 @@ private:
    
    void aboutToShow();
    void setupDialog();
+   void keyPressEvent( QKeyEvent *ev);
 
 private:           
    class KScoreDialogPrivate;
