cvs server: Diffing .
Index: newgamedlg.cpp
===================================================================
RCS file: /home/kde/kdegames/kpoker/newgamedlg.cpp,v
retrieving revision 1.7
diff -u -p -r1.7 newgamedlg.cpp
--- kdegames/kpoker/newgamedlg.cpp	2000/12/26 02:33:19	1.7
+++ kdegames/kpoker/newgamedlg.cpp	2002/03/04 09:00:55
@@ -25,6 +25,7 @@
 // KDE includes
 #include <klocale.h>
 #include <knuminput.h>
+#include <knumvalidator.h>
 #include <kapp.h>
 #include <kconfig.h>
 
@@ -69,6 +70,8 @@ NewGameDlg::NewGameDlg(QWidget* parent) 
  l = new QHBoxLayout(topLayout);
  l->addWidget(new QLabel(i18n("Start money of the players"), plainPage()));
  moneyOfPlayers = new QLineEdit(QString("%1").arg(money), plainPage());
+ moneyOfPlayers->setValidator( new KIntValidator( 0,999999,moneyOfPlayers ) );
+
  l->addWidget(moneyOfPlayers);
 
  l = new QHBoxLayout(topLayout);
cvs server: Diffing sounds
