--- kdegames-2.1/kpat/pwidget.cpp.bug	Sun Feb 18 16:29:19 2001
+++ kdegames-2.1/kpat/pwidget.cpp	Thu Feb 22 14:21:31 2001
@@ -298,15 +298,10 @@
     if(!pm.isNull()) {
         cardMap::self()->setBackSide(pm);
         config->writeEntry("Back", deck);
-        bool ret = cardMap::self()->setCardDir(cards);
-        if (!ret) {
-            config->writeEntry("Back", "");
-
-        }
+    } else if(!deck.isEmpty())
+        KMessageBox::sorry(this, i18n("Could not load background image!"));
+    if(cardMap::self()->setCardDir(cards))
         config->writeEntry("Cards", cards);
-    } else
-        KMessageBox::sorry(this,
-                           i18n("Could not load background image!"));
 
     if (dill) {
         dill->canvas()->setAllChanged();
